src/ReTestItems.jl

```diff
@@ -428,7 +431,7 @@ end
 
 any_non_pass(ts::DefaultTestSet) = ts.anynonpass
 
-function record_timeout!(testitem, run_number::Int, timeout_limit::Real)
+function record_timeout!(testitem, run_number::Int, timeout_limit::Float64)
     timeout_s = round(Int, timeout_limit)
```

Comment:
yah, this is a hassle. it's just because we were lax before and so were allowing users to pass any `Real`, so in this PR i converted them to Float64 asap, so we at least know the type we have... but i think going the other way and rounding them to Int is the better idea (we still need to accept Floats to avoid a breaking change, as you say)